acm/src/main/java/org/literacybridge/acm/store/AudioItemIndex.java

```diff
@@ -380,6 +361,24 @@ private Playlist getPlaylist(final String uuid, final Playlist playlist) throws
         }
     }
 
+    private void loadPlaylistFromPostingList(LeafReader leafReader, String playlistUuid, Playlist.Builder playlistBuilder) throws IOException {
+        PostingsEnum tp = leafReader.postings(new Term(PLAYLISTS_FIELD, playlistUuid), PostingsEnum.PAYLOADS);
```

Comment:
Even after several readings, I can't puzzle any meaning out of "tp". What's this variable's use? Can you make a name out of that?